block-storage-attacher/tests/e2e/conf/prod-us-east.conf

```diff
@@ -41,18 +41,18 @@ tor01_PVG_CRUISER_PRIVATE_VLAN=2259903
 
 # For plugin build
 GOPATH=$GOPATH  					# Read from environment
-BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/block-storage-attacher/tests/e2e/
+BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/
 
 # Latest Storage Plugin images to be used for e2e testing
 IMAGE_REGISTRY=registry.ng.bluemix.net
 USER_NAMESPACE=blockvolume
 PLUGIN_IMAGE=armada-block-volume-attacher
-PLUGIN_BUILD=latest
+PLUGIN_BUILD=$PLUGIN_BUILD
 
 # Helm chart install
 KUBECONFIG=$KUBECONFIG			# Read from environment(incase of dev executions)
 HELM_CHART_REPO=$HELM_CHART_REPO	
-HELM_CHART=$GOPATH/src/github.ibm.com/alchemy-containers/block-volume-attacher/helm/ibmcloud-block-storage-attacher/
+HELM_CHART=$GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/helm/ibmcloud-block-storage-attacher/
```

Comment:
This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.